Wordle.Api/Wordle.Api/Wordle.Api/Controllers/WordController.cs

```diff
@@ -26,5 +26,33 @@ public async Task<string> GetWordOfTheDay(double offsetInHours, DateTime? date =
             return await _wordService.GetWordOfTheDay(TimeSpan.FromHours(offsetInHours), date);
         }
 
+        [HttpGet("AllWords")]
+        public async Task<string[]> GetAllWords() 
+        {
+            return await _wordService.GetAllWords();
+        }
+
+        [HttpGet("AllIsCommon")]
+        public async Task<string[]> GetAllIsCommon()
+        {
+            return await _wordService.GetAllIsCommon();
+        }
+
+        [HttpPost("AddWord")]
+        public async Task<Word> AddWord(string word, bool isCommon) 
+        {
+            return await _wordService.AddWord(word, isCommon);   
+        }
+        [HttpPost("UpdateWord")]
+        public async Task<Word?> UpdateWord(string word, bool isCommon)
+        {
+            return await _wordService.UpdateWord(word, isCommon);
+        }
+
+        [HttpPost("DeleteWord")]
+        public async Task<Word?> DeleteWord(string word)
+        {
+            return await _wordService.DeleteWord(word);
+        }
```

Comment:
For update word, you could do an HttpPut and then for the delete you could use HttpDelete, it's just more standard. And I think the name is all you need to change there and you should be good! It'll indicate in Swagger if it was changed, the delete will be red and I think the update will be orange if I remember correctly.

Reply:
Noted! I'll look into implementing that!